.gitignore

```diff
@@ -1,3 +1,4 @@
 admin/config/
 shared/config/
 public/config/
+.idea
```

Comment:
You can create a global .gitignore file, which is a list of rules for ignoring files in every Git repository on your computer. For example, you might create the file at ~/.gitignore_global and add some rules to it.
https://help.github.com/articles/ignoring-files/